Upstream-Status: Pending

Avoids these kind of warnings:
Warning: 'libgnutls.la': linking libtool libraries using a non-POSIX archiver requires 'AM_PROG_AR' in 'configure.ac'

Signed-Off-by: Nitin A Kamble <nitin.a.kamble@intel.com>
Date: 2012/07/09
Index: gnutls-2.12.19/configure.ac
===================================================================
--- gnutls-2.12.19.orig/configure.ac
+++ gnutls-2.12.19/configure.ac
@@ -28,6 +28,7 @@ AC_CONFIG_MACRO_DIR([m4])
 
 AM_INIT_AUTOMAKE([1.10 no-dist-gzip dist-bzip2 -Wall -Wno-override])
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
+m4_ifdef([AM_PROG_AR], [AM_PROG_AR]) # required by automake 1.12
 AM_CONFIG_HEADER(config.h)
 
 AC_MSG_RESULT([***
Index: gnutls-2.12.19/lib/configure.ac
===================================================================
--- gnutls-2.12.19.orig/lib/configure.ac
+++ gnutls-2.12.19/lib/configure.ac
@@ -28,6 +28,7 @@ AC_CONFIG_MACRO_DIR([m4])
 
 AM_INIT_AUTOMAKE([1.10 -Wall -Wno-override])
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
+m4_ifdef([AM_PROG_AR], [AM_PROG_AR]) # required by automake 1.12
 AM_CONFIG_HEADER(config.h)
 
 AC_PROG_CC
Index: gnutls-2.12.19/libextra/configure.ac
===================================================================
--- gnutls-2.12.19.orig/libextra/configure.ac
+++ gnutls-2.12.19/libextra/configure.ac
@@ -28,6 +28,7 @@ AC_CONFIG_MACRO_DIR([m4])
 
 AM_INIT_AUTOMAKE([1.10 -Wall -Wno-override])
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
+m4_ifdef([AM_PROG_AR], [AM_PROG_AR]) # required by automake 1.12
 AM_CONFIG_HEADER(config.h)
 
 AC_PROG_CC
